packages/frontend/src/pages/Token/Communities.tsx

```diff
@@ -0,0 +1,87 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useCallback } from 'react'
+import { useForm } from 'react-hook-form'
+import { PrimaryButton } from 'src/components/Button'
+import Input from 'src/components/Input'
+import Box from 'src/theme/components/Box'
+import { Column } from 'src/theme/components/Flex'
+import * as Text from 'src/theme/components/Text'
+import { z } from 'zod'
+
+import * as styles from './style.css'
+
+// zod schemes
+const schema = z.object({
+    website: z.string().url().optional(),
+    telegram: z.string().regex(/^https?:\/\/(t\.me|telegram\.me)\/[a-zA-Z0-9_]{5,}$/),
+    twitter: z.string().regex(/^https?:\/\/(twitter\.com|x\.com)\/[a-zA-Z0-9_]{1,15}$/),
+    discord: z.string().regex(/^https?:\/\/(discord\.gg|discord\.com\/invite)\/[a-zA-Z0-9-]+$/),
+})
+
+export default function Communities() {
+    const {
+        register,
+        handleSubmit,
+        setValue,
+        formState: { errors },
+    } = useForm<z.infer<typeof schema>>({
+        resolver: zodResolver(schema),
+    })
+
+    const setCommunities = useCallback(async (data: z.infer<typeof schema>) => {
+        console.log('data: ', data)
+    }, [])
+
+    return (
+        <Column gap="32">
+            <Text.Custom color="text2" fontWeight="normal" fontSize="24">
+                Website and Communities
+            </Text.Custom>
+            <Column as="form" onSubmit={handleSubmit(setCommunities)} gap="24">
```

Comment:
There's bunch of formatting errors in this file, can you fix these? You can just run `yarn lint:fix` and it will automatically fix it.